### PR TITLE
fix(POEditor.com): update pl.po 

### PR DESCRIPTION
**Pull request by vomikan**
This change is [Reviewable](https://reviewable.io/reviews/moneymanagerex/moneymanagerex/1925)

### CODE REVIEW

**Comment by GitCop** [comment]
Thanks for your pull request while there were the following issues

 * Commit: 1fbb7c1f2ecce0629da2b5050e8c491b9fa41607
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert, cleanup, format, doc, clean**

Guidelines are available at https://github.com/moneymanagerex/moneymanagerex/wiki/GitCop

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Thanks for your pull request while there were the following issues

 * Commit: 1fbb7c1f2ecce0629da2b5050e8c491b9fa41607
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert, cleanup, format, doc, clean**

Guidelines are available at https://github.com/moneymanagerex/moneymanagerex/wiki/GitCop

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Thanks for your pull request while there were the following issues

 * Commit: 1fbb7c1f2ecce0629da2b5050e8c491b9fa41607
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert, cleanup, format, doc, clean**

 * Commit: d47a8f126b49a8ca45b4d483ebe09d999bac6b62
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert, cleanup, format, doc, clean**

Guidelines are available at https://github.com/moneymanagerex/moneymanagerex/wiki/GitCop

---
This message was auto-generated by https://gitcop.com

File: po/pl.po [hunk 1]
```diff
@@ -1,29 +1,20 @@
 msgid ""
 msgstr ""
-"Project-Id-Version: Money Manager Ex\n"
-"Report-Msgid-Bugs-To: developer@moneymanagerex.org\n"
-"POT-Creation-Date: 2019-01-11 00:58+0100\n"
-"PO-Revision-Date: 2018-08-29 15:40+0200\n"
-"Last-Translator: Tomasz Słodkowicz\n"
```
Comment (slodki) [inline po/pl.po]: Why we remove headers from po files?

File: po/pl.po [hunk 2]
```diff
@@ -238,12 +218,8 @@ msgid "Enter any contact information for the financial institution."
 msgstr "Wprowadź dowolne dane kontaktowe instytucji finansowej."
 
 #: src/accountdialog.cpp:391
-msgid ""
-"Enter any login/access information for the financial institution. This is "
-"not secure as anyone with access to the mmb file can access it."
-msgstr ""
-"Wprowadź informacje o loginie/haśle dla instytucji finansowej. Nie jest to "
-"bezpieczne jeśli inni mogą mieć dostęp do pliku mmb."
+msgid "Enter any login/access information for the financial institution. This is not secure as anyone with access to the mmb file can access it."
```
Comment (slodki) [inline po/pl.po]: Can we set line length to 78 chars?

File: po/pl.po [hunk 3]
```diff
@@ -411,9 +382,8 @@ msgid "% Rate"
 msgstr "% amortyzacji"
 
 #: src/assetdialog.cpp:241
-msgid ""
-"Enter the rate at which the asset changes its value in percentage per year"
-msgstr "Wprowadź stopę zmiany wartości aktywów w procentach na rok"
+msgid "Enter the rate at which the asset changes its value in percentage per year"
+msgstr ""
```
Comment (slodki) [inline po/pl.po]: Broken here!

File: po/pl.po [hunk 4]
```diff
@@ -526,21 +494,12 @@ msgid "Confirm Asset Deletion"
...
-"MMEX allows you to track fixed assets like cars, houses, land and others. "
-"Each asset can have its value appreciate by a certain rate per year, "
-"depreciate by a certain rate per year, or not change in value. The total "
-"assets are added to your total financial worth."
-msgstr ""
-"MMEX umożliwia śledzenie wartości środków trwałych, takich jak samochody, "
-"domy, grunty i inne. Każdy element może mieć wartość obniżaną "
-"(amortyzowaną) o określony procent rocznie lub nie zmienianą. Suma wartości "
-"aktywów jest dodawana do całkowitej wartości finansowej."
+msgid "MMEX allows you to track fixed assets like cars, houses, land and others. Each asset can have its value appreciate by a certain rate per year, depreciate by a certain rate per year, or not change in value. The total assets are added to your total financial worth."
```
Comment (slodki) [inline po/pl.po]: Lines too long

File: po/pl.po [hunk 5]
```diff
@@ -608,34 +566,30 @@ msgid "Enter any string to find related assets"
...
-msgid ""
-"Account \t Date\t   Value\n"
+msgid "Account \t Date\t   Value\n"
 "\n"
-msgstr ""
-"Konto \t Data\t   Watość\n"
+""
+msgstr "Konto \t Data\t   Watość\n"
 "\n"
+""
```
Comment (slodki) [inline po/pl.po]: empty line added?

File: po/pl.po [hunk 6]
```diff
@@ -918,7 +862,7 @@ msgstr "Powtórzenia"
 
 #: src/billsdepositsdialog.cpp:461
 msgid "Advance the next occurring date with the specified values"
-msgstr "Przestaw datę następnego wystąpienia w oparciu o ustaloną wartość"
+msgstr ""
```
Comment (slodki) [inline po/pl.po]: Broken here

File: po/pl.po (hunk 1 above)
Comment (vomikan) [reply po/pl.po]: That poeditor.com stupid specific.

**Comment by slodki** [comment]
Already included in #1922

**Pull request closed without merging**